src/web3-utils.js

```diff
@@ -106,5 +106,9 @@ export function getUnknownBalance() {
   return new BN('-1')
 }
 
+export function isValidEnsName(name) {
+  return /^([a-z0-9]+\.)+eth$/.test(name)
```

Comment:
This regex is too limiting as ENS names can have any unicode character. From the documentation:

> **8.16 What about foreign characters? What about upper case letters? Is any unicode character valid?**
Since the ENS contracts only deal with hashes, they have no direct way to enforce limits on what can be registered; character length restrictions are implemented by allowing users to challenge a short name by providing its preimage to prove it’s too short.
This means that you can in theory register both ‘foo.eth’ and ‘FOO.eth’, or even <picture of my cat>.eth. However, resolvers such as browsers and wallets should apply the nameprep algorithm to any names users enter before resolving; as a result, names that are not valid outputs of nameprep will not be resolvable by standard resolvers, making them effectively useless. DApps that assist users with registering names should prevent users from registering unresolvable names by using nameprep to preprocess names being requested for registration.
https://media.readthedocs.org/pdf/ens/latest/ens.pdf

I would at least allow common special chars like `-`, `_`, `@` and uppercase letters, and we can decide how to securely roll out support for other alphabets down the line.

Reply:
I mean, yeah, if we _really_ want to support all possible ENS names, we could use something like `[a-z0-9\\u00a1-\\uffff]` to satisfy unicode names.

Normal users wouldn't be able to create organizations with these types of names, since our onboarding is fixed to `[a-z0-9]` but I _guess_ if you _really_ wanted to...

Reply:
But users (as ourselves with `open-apm.aragonid.eth`) can create those names by interacting with aragon-id directly or creating their DAO using the kit but not our onboarding. I'd say let's just add some commonly used chars

Reply:
Opened it up so the allowed set of characters is `[a-zA-Z0-9-]`; most other "special" delimiting characters (e.g. `@`, `!`, `#`, etc. are not allowed because they're not allowed in IRIs).